<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataImageConfigDialog</class>
 <widget class="QDialog" name="DataImageConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>498</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A larger system size is recommended because you can install more apps.</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>38</y>
     <width>179</width>
     <height>83</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Choose system size:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="text">
     <string>8GB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="text">
     <string>16GB</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3">
    <property name="text">
     <string>32GB(Recommended)</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
